config/application.php

```diff
@@ -89,6 +89,14 @@
 Config::define('SCRIPT_DEBUG', false);
 ini_set('display_errors', 0);
 
+/**
+ * Allow Wordpress to detect HTTPS when used behind a reverse proxy or a load balancer
```

Comment:
Hey there, can we correct to "WordPress" please? https://codex.wordpress.org/Function_Reference/capital_P_dangit 😝

Reply:
done